lib/api/controllers/securityController.js

```diff
@@ -125,8 +150,8 @@ class SecurityController {
 
     return this.kuzzle.repositories.role.loadMultiFromDatabase(request.input.body.ids)
       .then(roles => {
-        roles = roles.map(role => formatProcessing.formatRoleForSerialization(role));
-        return {hits: roles, total: roles.length};
+        const formatted = roles.map(role => formatProcessing.formatRoleForSerialization(role));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 95 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)